doc/cdi-populators.md

```diff
@@ -0,0 +1,99 @@
+# CDI Volume Populators
+
+## Recommended knowledge
+
+What are [populators](https://kubernetes.io/blog/2022/05/16/volume-populators-beta/)
+
+## Introduction
+CDI volume populators are CDIs implementation of populating PVCs by importing/uploading/cloning data utilizing the new `dataSourceRef` field. New controllers and custom resources for each population method were introduced.
```

Comment:
Don't know if this is incorrect but my spanish-talking brain tells me to change `populating` for a noun in `implementation of populating PVCs`.

Maybe `CDI volume populators are CDI's implementation for the population of PVCs by...` sounds good too.